docs/data/userFriends.md

```diff
@@ -0,0 +1,20 @@
+
+##userFriends - links users on a many-to-many relationship ( jonction table/model )
+[https://en.wikipedia.org/wiki/Associative_entity](https://en.wikipedia.org/wiki/Associative_entity)
+
+Because a user can have many other users as firends, and at the same time he can be one of many friends of a certain user.
+
+v 0.1.0
+
+I wansn't sure if we had to create this table or if seequelize will crefate it automatically for us, but it looks like is best if we do create it ourselves. 
+[https://sequelize.org/docs/v6/core-concepts/assocs/#many-to-many-relationships](https://sequelize.org/docs/v6/core-concepts/assocs/#many-to-many-relationships)
+[https://sequelize.org/docs/v6/advanced-association-concepts/advanced-many-to-many/](https://sequelize.org/docs/v6/advanced-association-concepts/advanced-many-to-many/)
+
```

Comment:
Sequelize will create it, but documenting it is useful. Please see the below repo for an example of a through table

https://github.com/mz106/mysql-through-table-example